ChatQnA/README.md

```diff
@@ -86,28 +86,21 @@ Now the TGI Gaudi will launch the FP8 model by default. Please note that current
 
 ## Launch Redis
 ```bash
-docker pull redis/redis-stack:latest
 docker compose -f langchain/docker/docker-compose-redis.yml up -d
 ```
 
 ## Launch LangChain Docker
 
-### Build LangChain Docker Image
-
-```bash
-cd langchain/docker/
-bash ./build_docker.sh
-```
-
-### Lanuch LangChain Docker
-
 Update the `HUGGINGFACEHUB_API_TOKEN` environment variable with your huggingface token in the `docker-compose-langchain.yml`
 
 ```bash
 docker compose -f docker-compose-langchain.yml up -d
```

Comment:
Need to add "cd langchain/docker/", otherwise can't find the docker-compose-langchain.yml file. Or change the command to "docker compose -f langchain/docker/docker-compose-langchain.yml up -d"